pkg/auth/rbac.go

```diff
@@ -185,6 +191,31 @@ func ValidateRbacPermission(line string) (p Permission, err error) {
 	}, nil
 }
 
+func ValidateTeamRbacPermission(line string) (team string, users []string, err error) {
+
+	p := strings.Split(line, ",")
```

Comment:
Can you rename this `p` variable to something that indicates what this variable holds?